IAR - 419: Unity audio streaming to ISAR

holo-istemi opened the pull request:
- Replaced AudioDeviceWindowsCore by AudioDeviceDummy in server endpoint so that system audio & microphone does not get involved to any audio at all on WebRTC level.

holo-istemi committed:
IAR-419: feat: Infrastructural changes for Unity specific Audio

- Replaced AudioDeviceWindowsCore by AudioDeviceDummy in server endpoint so that system audio & microphone does not get involved to any audio at all on WebRTC level.

holo-istemi committed:
IAR-419: chore: Submodule Update

holo-istemi committed:
Merge branch 'dev/m71' into IAR-419

aidanhowie19 reviewed (changes requested):
As discussed in the submodule, these changes should be reverted and we should use the methods to avoid changing the WebRTC codebase

aidanhowie19 committed:
Submodule webrtc/xplatform/webrtc updated

Submodule webrtc/xplatform/webrtc:
    > 91b2888b0f - IAR-104: refactor: Revert changes to webrtc codebase

aidanhowie19 committed:
Submodule webrtc/xplatform/webrtc updated

Submodule webrtc/xplatform/webrtc:
    > 7b1c5cb2d4 - IAR-104: chore: cleanup
    > fcc4240928 - IAR-104: fix: vscode highlighting
    > 350c1644b5 - IAR-438: perf: Set thread names to be seen in perf tools
    > dd6694a751 - IAR-399: build: Tasks to include generation of flatbuffers extensions

aidanhowie19 committed:
Merge remote-tracking branch 'origin/IAR-393' into IAR-419

aidanhowie19 committed:
Submodule webrtc/xplatform/webrtc updated

Submodule webrtc/xplatform/webrtc:
    > 76befdaa89 - IAR-104: chore: Update intellisense paths
